go-chaos/internal/logging.go

```diff
@@ -0,0 +1,39 @@
+// Copyright 2022 Camunda Services GmbH
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package internal
+
+import (
+	"fmt"
+)
+
+func VerbosityLogging(text string, a ...any) {
```

Comment:
:upside_down_face: names are too long IMO, I'd either move them to a new package called `log` so calls can look like `log.Verbose(...)` and `log.Info(...)`, or leave them in the `internal` package but rename the function to `LogInfo(...)` and `LogVerbose(...)` or similar.

Reply:
Yeah was also not sure about them. I didn't wanted to conflict to much with the normal log package and functions